sopel/coretasks.py

```diff
@@ -1,9 +1,17 @@
 # coding=utf-8
-"""Tasks that allow the bot to run, but aren't user-facing functionality
+"""Core Sopel plugin that listen to basic IRC messages.
+
+This plugin allows the bot to run without user-facing functionality:
+
+* it handles client capability negotiation,
+* it handles client auth (both nick auth and server auth),
+* it handles connection registration (RPL_WELCOME, RPL_LUSERCLIENT), dealing
+  with error cases such as nick already in use
+* it tracks known channels & users (join, quit, nick change and other events).
```

Comment:
```suggestion
* it tracks known channels & users (join, quit, nick change and other events)
* it manages blocked (ignored) users
```

Until/unless the `.blocks` stuff gets moved out of this file, it should get mentioned here.

Reply:
I knew I forgot something.